src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb

```diff
@@ -74,13 +74,59 @@ namespace N$$.P
                                 <Document>$$</Document>,
                                 showCompletionInArgumentLists:=showCompletionInArgumentLists)
 
-                state.SendTypeChars("us")
+                state.SendTypeChars("usi")
```

Comment:
did you file issue about this?

Reply:
oh wait, can we undo this?  it looks like you handle this case and preselect `using`.  so this test change shouldn't be necessary.

Reply:
> it looks like you handle this case and preselect using

We do not do this, and it's questionable whether we will do this in the future. For now, the test change cannot be reverted.

Reply:
why is it questionable?

Reply:
also, isn't this a one line fix?  why not just fix it during the PR to fix completion issues?

Reply:
➡️ Moved discussion to #44459